requirements.txt

```diff
@@ -1,5 +1,5 @@
 numpy>=1.22
-networkx>3.0
+networkx>=2.8,<3.0
```

Comment:
what's the reason for this?

Reply:
It is due to mentpy which requires networkx >=2.5,<3.0

Reply:
why do it in main requirement and not in docs/requirements? Doesn't make sense to change graphix's dependency for mentpy requirements

Reply:
Sorry, it was my mistake. I will be careful to keep dependencies separable from the main implementation from the next time.